dags/pytorch_xla/demo.py

```diff
@@ -0,0 +1,24 @@
+import datetime
```

Comment:
Nit: header for the file,

Reply:
Oops, this file is from a live demo I did a few weeks ago. Will remove.